src/widgets/Notebook.cpp

```diff
@@ -1006,6 +1007,29 @@ SplitNotebook::SplitNotebook(Window *parent)
                                        [this](SplitContainer *sc) {
                                            this->select(sc);
                                        });
+
+    this->signalHolder_.managedConnect(
+        getApp()->windows->scrollToMessageSignal,
+        [this](const MessagePtr &message) {
+            for (auto &&item : this->items())
+            {
+                if (auto sc = dynamic_cast<SplitContainer *>(item.page))
+                {
+                    auto &&splits = sc->getSplits();
+                    auto split = std::find_if(
+                        splits.begin(), splits.end(), [message](Split *split) {
+                            return split->getChannel()->getType() !=
+                                       Channel::Type::TwitchMentions &&
+                                   split->getChannelView().scrollToMessage(
+                                       message);
```

Comment:
Convert this to a for loop instead, remember you'll need to return to use `return` since `break` will only break the inner loop

Reply:
Done in 4798ed67d9a77be42a2afd01c74bc3a58cacee37.